server/controllers/admin/fundingSources.js

```diff
@@ -0,0 +1,73 @@
+const db = require('../../lib/db');
+
+module.exports = {
+  create,
+  read,
+  update,
+  detail,
+  delete : remove,
+};
+
+// add a new tag
+function create(req, res, next) {
+  const sql = `INSERT INTO funding_source SET ?`;
+  const data = req.body;
+  data.uuid = data.uuid ? db.bid(data.uuid) : db.uuid();
+  db.exec(sql, data)
+    .then(() => {
+      res.sendStatus(201);
```

Comment:
Could you make these CRUD APIs return the same values as the rest of the BHIMA CRUD APIs?

https://github.com/Third-Culture-Software/bhima/wiki/Design:-Server-CRUD-APIs

Create should return the `id`/`uuid` of the resource created.